src/explorer/webview.ts

```diff
@@ -346,9 +346,8 @@ export class ExplorerWebview {
             if (options.tooltip === undefined) {
                 if (options.message !== undefined) {
                     options.tooltip = options.message;
-                } else if (options.location.uri !== undefined) {
-                    // @ts-ignore external exist on the webview side
-                    options.tooltip = options.location.uri.external.replace("%3A", ":");
+                } else if (options.location.uriAsString !== undefined) {
```

Comment:
>.location.uriAsString !== undefined [](start = 34, length = 35)

Why does below just use options.location.uriAsString?

Reply:
They should indeed both be checking for === undefined.

---
In reply to: [413326716](https://github.com/microsoft/sarif-vscode-extension/pull/257#discussion_r413326716) [](ancestors = 413326716)